.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+jobs:
+  test-job:
+    docker:
+      - image: circleci/node:8
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+
+      - run:
+          name: Install less compiler
+          command: sudo npm install less@3.0.1 -g
+
+      - run:
+          name: Install node dependencies
+          command: npm install
+
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+
+      - run:
+          name: Run frontend tests
+          command: npm run build-css && npm test
+
+  build:
+    docker:
+      - image: circleci/node:8
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+
+      - run:
+          name: Install node dependencies
+          command: npm install
+
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+
+      - run:
+          name: Build app
+          command: |
+            npm run build
```

Comment:
It doesn't, but it might be easier for future maintenance? Similar to "always use brackets around conditional bodies for the next guy" sort of arguments

Reply:
I feel like there's no likelihood of error here like there is for conditional bodies. I also don't feel too strongly about it, just consider the lone pipe ugly as sin :)

Reply:
I don't have a strong opinion on this one way or the other, just trying to understand your take on this @Shadowfiend. 

I noticed that the ["how to version up to circle v2" docs](https://circleci.com/docs/2.0/migrating-from-1-2/#interpolating-environment-variables) leave a pipe at the end of the line - so I would've, in my newness to circle, done it this way.  

Is the issue just that there's only one command here, where in the docs example (and line 63 below) there's a block of commands so it's ok there? or is there preferred way to handle those cases as well?

Reply:
Correct! The pipe is YAML for “there are multiple lines of stuff after this and they all belong in this property”, and it's unnecessary if there's only one line of stuff.

To be clear, my complaint was 100% aesthetic, though, not functional at all.

Reply:
gotcha, thanks!